prow/scripts/cluster-integration/helpers/generate-ark-config.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+
+shout "Create a Secret for Ark"
+
+export SA_NAME=sa-gcs-plank
+export SA_DISPLAY_NAME=sa-gcs-plank
+export SECRET_FILE=sa-gcs-plank
+export ROLE=roles/storage.objectAdmin
+
+gcloud iam service-accounts create $SA_NAME --display-name $SA_DISPLAY_NAME
```

Comment:
why sa is created here? Have a look how `preset-sa-kyma-artifacts` is used. If you need assistance please ping me

Reply:
Service account name
sa-kyma-backup-restore

Service account ID for @kyma-project.iam.gserviceaccount.com
sa-kyma-backup-restore

service account email address
sa-kyma-backup-restore@kyma-project.iam.gserviceaccount.com 

bucket:
kyma-backup-restore

Link for gsutil 
gs://kyma-backup-restore



  - labels:
      preset-kyma-backup-restore-bucket: "true"
    env:                                       
      - name: KYMA_BACKUP_RESTORE_BUCKET       
        value: gs://kyma-backup-restore     

Question, do I need to add a new label in config.yaml for the service account I have created in google cloud?
```
  - labels:
      preset-sa-kyma-backup-restore: "true"
    env:
    - name: GOOGLE_APPLICATION_CREDENTIALS
      value: /etc/credentials/sa-kyma-backup-restore/service-account.json
    volumes:
    - name: sa-kyma-backup-restore
      secret:
        secretName: sa-kyma-backup-restore
    volumeMounts:
    - name: sa-kyma-backup-restore
      mountPath: /etc/credentials/sa-kyma-backup-restore
```
I did it for the bucket.
```
  - labels:
      preset-kyma-backup-restore-bucket: "true"
    env:                                       
      - name: KYMA_BACKUP_RESTORE_BUCKET       
        value: gs://kyma-backup-restore    
```